Make it easier to set values for Image/File fields

### Is your feature request related to a problem? Please describe.

It's extremely tedious to set values for Image and File fields.  As reported by a customer: https://dotcms.zendesk.com/agent/tickets/103626

```
I find the Image Field uploading experience not convenient and I think the UI needs a little update.
There's one particular point that I can't stand every time I have to upload an image and it is that it is
not pointing by default to the host I am under in dotCMS. Since we need to upload the images in
the actual sites, then I think it's only normal to point to its directory right away.

Right now I have to go through the whole list of 100+ sites and open the folders manually.
Imagine uploading or updating 20 contentlet Images that way.

I have to reselect my host directory, open the little arrow to view the subfolders, click on asset and
select my image. This is a very long process.
```

### Describe the solution you'd like

There are several ways we could make this process (much) easier.  Here are my suggestions (in order of the value I think they'd have for customers):

1. Have dotCMS remember the last selected folder.
    * This might take a cookie, or might require some other info to be stored in the browser session.
        * But if the value stored was an identifier, keeping it in a cookie would present no privacy or security implications.
2. Create a "Favorites" (or equivalent) branch in the site selection tree.
    * Allow users to mark folders as favorites, to get them to show up in this list.
    * This could actually be the best overall solution (since it has value beyond just Image/File fields).
        * But it's probably a heavier lift than just remembering the last selected folder.
    * Note: This could possibly be done without too much work by adding "Favorite" as a type of Menu Link choice.
3. By default, select the current site (and open it in the navigation tree).
4. Allow specification of a default folder for Image fields and File fields.
    * Probably via a configuration property.

### Describe alternatives you've considered

* Image/File fields can not be imported or exported.
* So, the only real alternative right now appears to be to update the values using an external script making REST API calls.

However if a set the `defaultPath` variable as an inexistent folder and I try to add the content the server shows this error on modal and throws this exception:
![image](https://user-images.githubusercontent.com/64026161/145323757-f9b4e1c5-cbeb-4103-9224-5064a7807308.png)

```
08-Dec-2021 19:36:19.253 WARNING [url:GET//demo.dotcms.com/api/v1/workflow/schemes | lang:1 | ip:0:0:0:0:0:0:0:1 | Admin:false | start:12-08-2021 04:14:09 CST  ref:http://localhost:8080/c/portal/layout?p_l_id=71b8a1ca-37b6-4b6e-a43b-c7482f28db6c&p_p_id=content&p_p_action=0&&dm_rlout=1&r=1639012272243&in_frame=true&frame=detailFrame&container=true&angularCurrentPortlet=content  ?showArchive=false&contentTypeId=70579bcafb5e6fc2f7ec7f510d65f835] org.glassfish.jersey.servlet.WebComponent.filterFormParameters A servlet request to the URI http://localhost:8080/api/v1/workflow/schemes?showArchive=false&contentTypeId=70579bcafb5e6fc2f7ec7f510d65f835 contains form parameters in the request body but the request body has been consumed by the servlet or a servlet filter accessing the request parameters. Only resource methods using @FormParam will work as expected. Resource methods consuming the request body by other means will not work as expected.
19:36:25.843  ERROR model.Folder - identifier is null
com.dotmarketing.business.DotStateException: identifier is null
	at com.dotmarketing.business.IdentifierFactoryImpl.loadFromDb(IdentifierFactoryImpl.java:198) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.dotmarketing.business.IdentifierFactoryImpl.find(IdentifierFactoryImpl.java:462) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.dotmarketing.business.IdentifierAPIImpl.find_aroundBody4(IdentifierAPIImpl.java:81) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.dotmarketing.business.IdentifierAPIImpl$AjcClosure5.run(IdentifierAPIImpl.java:1) ~[dotcms_21.12_9aa6ca2.jar:?]
	at org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149) ~[aspectjrt-1.9.2.jar:?]
	at com.dotcms.aspects.aspectj.AspectJDelegateMethodInvocation.proceed(AspectJDelegateMethodInvocation.java:42) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.dotcms.aspects.interceptors.CloseDBIfOpenedMethodInterceptor.invoke(CloseDBIfOpenedMethodInterceptor.java:29) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.dotcms.aspects.aspectj.CloseDBIfOpenedAspect.invoke(CloseDBIfOpenedAspect.java:41) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.dotmarketing.business.IdentifierAPIImpl.find(IdentifierAPIImpl.java:80) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.dotmarketing.portlets.folders.business.FolderAPIImpl.findParentFolder_aroundBody4(FolderAPIImpl.java:164) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.dotmarketing.portlets.folders.business.FolderAPIImpl$AjcClosure5.run(FolderAPIImpl.java:1) ~[dotcms_21.12_9aa6ca2.jar:?]
	at org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:149) ~[aspectjrt-1.9.2.jar:?]
	at com.dotcms.aspects.aspectj.AspectJDelegateMethodInvocation.proceed(AspectJDelegateMethodInvocation.java:42) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.dotcms.aspects.interceptors.CloseDBIfOpenedMethodInterceptor.invoke(CloseDBIfOpenedMethodInterceptor.java:29) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.dotcms.aspects.aspectj.CloseDBIfOpenedAspect.invoke(CloseDBIfOpenedAspect.java:41) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.dotmarketing.portlets.folders.business.FolderAPIImpl.findParentFolder(FolderAPIImpl.java:162) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.dotmarketing.portlets.folders.model.Folder.getParentPermissionable(Folder.java:282) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.dotmarketing.portlets.browser.BrowserUtil.getFolderPathIds(BrowserUtil.java:332) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.dotmarketing.portlets.browser.BrowserUtil.getDefaultPathFolderPathIds(BrowserUtil.java:353) ~[dotcms_21.12_9aa6ca2.jar:?]
	at org.apache.jsp.html.portlet.ext.contentlet.field.edit_005ffield_jsp._jspService(edit_005ffield_jsp.java:1370) ~[?:?]
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:71) ~[jasper.jar:9.0.41]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733) ~[servlet-api.jar:4.0.FR]
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:477) ~[jasper.jar:9.0.41]
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:385) ~[jasper.jar:9.0.41]
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:329) ~[jasper.jar:9.0.41]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733) ~[servlet-api.jar:4.0.FR]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:712) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationDispatcher.doInclude(ApplicationDispatcher.java:580) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationDispatcher.include(ApplicationDispatcher.java:516) ~[catalina.jar:9.0.41]
	at org.apache.jasper.runtime.JspRuntimeLibrary.include(JspRuntimeLibrary.java:954) ~[jasper.jar:9.0.41]
	at org.apache.jsp.html.portlet.ext.contentlet.edit_005fcontentlet_jsp._jspService(edit_005fcontentlet_jsp.java:4699) ~[?:?]
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:71) ~[jasper.jar:9.0.41]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733) ~[servlet-api.jar:4.0.FR]
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:477) ~[jasper.jar:9.0.41]
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:385) ~[jasper.jar:9.0.41]
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:329) ~[jasper.jar:9.0.41]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733) ~[servlet-api.jar:4.0.FR]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:712) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationDispatcher.doInclude(ApplicationDispatcher.java:580) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationDispatcher.include(ApplicationDispatcher.java:516) ~[catalina.jar:9.0.41]
	at org.apache.jasper.runtime.JspRuntimeLibrary.include(JspRuntimeLibrary.java:954) ~[jasper.jar:9.0.41]
	at org.apache.jsp.html.portal.layout_005fportal_jsp._jspService(layout_005fportal_jsp.java:635) ~[?:?]
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:71) ~[jasper.jar:9.0.41]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733) ~[servlet-api.jar:4.0.FR]
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:477) ~[jasper.jar:9.0.41]
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:385) ~[jasper.jar:9.0.41]
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:329) ~[jasper.jar:9.0.41]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733) ~[servlet-api.jar:4.0.FR]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:712) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationDispatcher.doInclude(ApplicationDispatcher.java:580) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationDispatcher.include(ApplicationDispatcher.java:516) ~[catalina.jar:9.0.41]
	at com.liferay.portlet.PortletRequestDispatcherImpl.include(PortletRequestDispatcherImpl.java:206) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.liferay.portal.struts.PortletRequestProcessor.doInclude(PortletRequestProcessor.java:344) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.liferay.portal.struts.PortletRequestProcessor.doForward(PortletRequestProcessor.java:316) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.dotcms.repackage.org.apache.struts.tiles.TilesRequestProcessor.processTilesDefinition(TilesRequestProcessor.java:239) ~[dot.struts-1.2.10_1.jar:?]
	at com.dotcms.repackage.org.apache.struts.tiles.TilesRequestProcessor.processForwardConfig(TilesRequestProcessor.java:302) ~[dot.struts-1.2.10_1.jar:?]
	at com.dotcms.repackage.com.oroad.stxx.plugin.StxxTilesRequestProcessor.processForwardConfig(StxxTilesRequestProcessor.java:134) ~[dot.stxx-1.3_3.jar:?]
	at com.liferay.portal.struts.PortletRequestProcessor.processForwardConfig(PortletRequestProcessor.java:203) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.dotcms.repackage.org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:241) ~[dot.struts-1.2.10_1.jar:?]
	at com.liferay.portal.struts.PortletRequestProcessor.process(PortletRequestProcessor.java:89) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.liferay.portlet.StrutsPortlet.include(StrutsPortlet.java:152) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.liferay.portlet.StrutsPortlet.doView(StrutsPortlet.java:104) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.dotcms.repackage.javax.portlet.GenericPortlet.doDispatch(GenericPortlet.java:235) ~[dot.portlet-1.0_2.jar:?]
	at com.dotcms.repackage.javax.portlet.GenericPortlet.render(GenericPortlet.java:163) ~[dot.portlet-1.0_2.jar:?]
	at com.liferay.portlet.ConcretePortletWrapper.render(ConcretePortletWrapper.java:114) ~[dotcms_21.12_9aa6ca2.jar:?]
	at org.apache.jsp.html.portal.layout_jsp._jspService(layout_jsp.java:630) ~[?:?]
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:71) ~[jasper.jar:9.0.41]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733) ~[servlet-api.jar:4.0.FR]
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:477) ~[jasper.jar:9.0.41]
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:385) ~[jasper.jar:9.0.41]
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:329) ~[jasper.jar:9.0.41]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733) ~[servlet-api.jar:4.0.FR]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:712) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationDispatcher.doInclude(ApplicationDispatcher.java:580) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationDispatcher.include(ApplicationDispatcher.java:516) ~[catalina.jar:9.0.41]
	at org.apache.jasper.runtime.JspRuntimeLibrary.include(JspRuntimeLibrary.java:954) ~[jasper.jar:9.0.41]
	at org.apache.jsp.html.portal.layout_005fportal_jsp._jspService(layout_005fportal_jsp.java:1611) ~[?:?]
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:71) ~[jasper.jar:9.0.41]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733) ~[servlet-api.jar:4.0.FR]
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:477) ~[jasper.jar:9.0.41]
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:385) ~[jasper.jar:9.0.41]
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:329) ~[jasper.jar:9.0.41]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733) ~[servlet-api.jar:4.0.FR]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53) ~[tomcat-websocket.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:712) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationDispatcher.processRequest(ApplicationDispatcher.java:459) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationDispatcher.doForward(ApplicationDispatcher.java:384) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationDispatcher.forward(ApplicationDispatcher.java:312) ~[catalina.jar:9.0.41]
	at com.liferay.portal.struts.StrutsUtil.forward(StrutsUtil.java:72) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.liferay.portal.struts.PortalRequestProcessor.doForward(PortalRequestProcessor.java:161) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.dotmarketing.struts.PortalRequestProcessor.doForward(PortalRequestProcessor.java:48) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.dotcms.repackage.org.apache.struts.tiles.TilesRequestProcessor.processTilesDefinition(TilesRequestProcessor.java:239) ~[dot.struts-1.2.10_1.jar:?]
	at com.dotcms.repackage.org.apache.struts.tiles.TilesRequestProcessor.processForwardConfig(TilesRequestProcessor.java:302) ~[dot.struts-1.2.10_1.jar:?]
	at com.dotcms.repackage.com.oroad.stxx.plugin.StxxTilesRequestProcessor.processForwardConfig(StxxTilesRequestProcessor.java:134) ~[dot.stxx-1.3_3.jar:?]
	at com.dotcms.repackage.org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:241) ~[dot.struts-1.2.10_1.jar:?]
	at com.liferay.portal.struts.PortalRequestProcessor.process(PortalRequestProcessor.java:147) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.dotcms.repackage.org.apache.struts.action.ActionServlet.process(ActionServlet.java:1202) ~[dot.struts-1.2.10_1.jar:?]
	at com.dotcms.repackage.org.apache.struts.action.ActionServlet.doGet(ActionServlet.java:412) ~[dot.struts-1.2.10_1.jar:?]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:626) ~[servlet-api.jar:4.0.FR]
	at com.liferay.portal.servlet.MainServlet.callParentService(MainServlet.java:244) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.liferay.portal.servlet.MainServlet.service(MainServlet.java:444) ~[dotcms_21.12_9aa6ca2.jar:?]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733) ~[servlet-api.jar:4.0.FR]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53) ~[tomcat-websocket.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at com.dotcms.repackage.com.liferay.filters.secure.SecureFilter.doFilter(SecureFilter.java:135) ~[dot.secure-filter-ukv_2.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at com.dotmarketing.filters.CMSFilter.doFilterInternal(CMSFilter.java:168) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.dotmarketing.filters.CMSFilter.doFilter(CMSFilter.java:49) ~[dotcms_21.12_9aa6ca2.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at com.dotcms.filters.interceptor.AbstractWebInterceptorSupportFilter.doFilter(AbstractWebInterceptorSupportFilter.java:90) ~[dotcms_21.12_9aa6ca2.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at com.dotcms.filters.interceptor.AbstractWebInterceptorSupportFilter.doFilter(AbstractWebInterceptorSupportFilter.java:90) ~[dotcms_21.12_9aa6ca2.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at com.dotcms.vanityurl.filters.VanityURLFilter.doFilter(VanityURLFilter.java:114) ~[dotcms_21.12_9aa6ca2.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at org.tuckey.web.filters.urlrewrite.RuleChain.handleRewrite(RuleChain.java:176) ~[urlrewritefilter-4.0.4.jar:4.0.4]
	at org.tuckey.web.filters.urlrewrite.RuleChain.doRules(RuleChain.java:145) ~[urlrewritefilter-4.0.4.jar:4.0.4]
	at org.tuckey.web.filters.urlrewrite.UrlRewriter.processRequest(UrlRewriter.java:92) ~[urlrewritefilter-4.0.4.jar:4.0.4]
	at org.tuckey.web.filters.urlrewrite.UrlRewriteFilter.doFilter(UrlRewriteFilter.java:389) ~[urlrewritefilter-4.0.4.jar:4.0.4]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at com.dotmarketing.filters.TimeMachineFilter.doFilter(TimeMachineFilter.java:140) ~[dotcms_21.12_9aa6ca2.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at com.dotmarketing.filters.ThreadNameFilter.doFilter(ThreadNameFilter.java:88) ~[dotcms_21.12_9aa6ca2.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at com.dotmarketing.filters.CookiesFilter.doFilter(CookiesFilter.java:50) ~[dotcms_21.12_9aa6ca2.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at com.dotmarketing.filters.CharsetEncodingFilter.doFilter(CharsetEncodingFilter.java:99) ~[dotcms_21.12_9aa6ca2.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at com.dotcms.filters.interceptor.AbstractWebInterceptorSupportFilter.doFilter(AbstractWebInterceptorSupportFilter.java:90) ~[dotcms_21.12_9aa6ca2.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at org.apache.catalina.filters.HttpHeaderSecurityFilter.doFilter(HttpHeaderSecurityFilter.java:126) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at com.dotcms.filters.NormalizationFilter.doFilter(NormalizationFilter.java:73) ~[dotcms_21.12_9aa6ca2.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97) ~[catalina.jar:9.0.41]
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143) ~[catalina.jar:9.0.41]
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92) ~[catalina.jar:9.0.41]
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690) ~[catalina.jar:9.0.41]
	at org.apache.catalina.valves.RemoteIpValve.invoke(RemoteIpValve.java:764) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78) ~[catalina.jar:9.0.41]
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343) ~[catalina.jar:9.0.41]
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:374) ~[tomcat-coyote.jar:9.0.41]
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65) ~[tomcat-coyote.jar:9.0.41]
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:888) ~[tomcat-coyote.jar:9.0.41]
	at org.apache.tomcat.util.net.Nio2Endpoint$SocketProcessor.doRun(Nio2Endpoint.java:1693) ~[tomcat-coyote.jar:9.0.41]
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49) ~[tomcat-coyote.jar:9.0.41]
	at org.apache.tomcat.util.net.AbstractEndpoint.processSocket(AbstractEndpoint.java:1122) ~[tomcat-coyote.jar:9.0.41]
	at org.apache.tomcat.util.net.Nio2Endpoint.setSocketOptions(Nio2Endpoint.java:335) ~[tomcat-coyote.jar:9.0.41]
	at org.apache.tomcat.util.net.Nio2Endpoint$Nio2Acceptor.completed(Nio2Endpoint.java:455) ~[tomcat-coyote.jar:9.0.41]
	at org.apache.tomcat.util.net.Nio2Endpoint$Nio2Acceptor.completed(Nio2Endpoint.java:391) ~[tomcat-coyote.jar:9.0.41]
	at sun.nio.ch.Invoker.invokeUnchecked(Invoker.java:127) ~[?:?]
	at sun.nio.ch.Invoker.invokeDirect(Invoker.java:158) ~[?:?]
	at sun.nio.ch.Invoker.invoke(Invoker.java:186) ~[?:?]
	at sun.nio.ch.UnixAsynchronousServerSocketChannelImpl.onEvent(UnixAsynchronousServerSocketChannelImpl.java:193) ~[?:?]
	at sun.nio.ch.KQueuePort$EventHandlerTask.run(KQueuePort.java:312) ~[?:?]
	at sun.nio.ch.AsynchronousChannelGroupImpl$1.run(AsynchronousChannelGroupImpl.java:112) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:834) [?:?]
19:36:25.883  ERROR portlet.PortletRequestDispatcherImpl - An exception occurred processing [/html/portlet/ext/contentlet/field/edit_field.jsp] at line [505]

502:         //IMAGE kind of field rendering
503:         else if (field.getFieldType().equals(
504:                 Field.FieldType.IMAGE.toString())) {
505:             final List<String> defaultPathFolderPathIds = BrowserUtil.getDefaultPathFolderPathIds(
506:                     contentlet, LegacyFieldTransformer.from(field),
507:                     user);
508:             defaultPathFolderPathIds.add(0, "root");


Stacktrace:
org.apache.jasper.JasperException: An exception occurred processing [/html/portlet/ext/contentlet/field/edit_field.jsp] at line [505]

502:         //IMAGE kind of field rendering
503:         else if (field.getFieldType().equals(
504:                 Field.FieldType.IMAGE.toString())) {
505:             final List<String> defaultPathFolderPathIds = BrowserUtil.getDefaultPathFolderPathIds(
506:                     contentlet, LegacyFieldTransformer.from(field),
507:                     user);
508:             defaultPathFolderPathIds.add(0, "root");


Stacktrace:
	at org.apache.jasper.servlet.JspServletWrapper.handleJspException(JspServletWrapper.java:626) ~[jasper.jar:9.0.41]
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:500) ~[jasper.jar:9.0.41]
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:385) ~[jasper.jar:9.0.41]
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:329) ~[jasper.jar:9.0.41]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733) ~[servlet-api.jar:4.0.FR]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:712) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationDispatcher.doInclude(ApplicationDispatcher.java:580) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationDispatcher.include(ApplicationDispatcher.java:516) ~[catalina.jar:9.0.41]
	at org.apache.jasper.runtime.JspRuntimeLibrary.include(JspRuntimeLibrary.java:954) ~[jasper.jar:9.0.41]
	at org.apache.jsp.html.portal.layout_005fportal_jsp._jspService(layout_005fportal_jsp.java:635) ~[?:?]
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:71) ~[jasper.jar:9.0.41]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733) ~[servlet-api.jar:4.0.FR]
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:477) ~[jasper.jar:9.0.41]
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:385) ~[jasper.jar:9.0.41]
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:329) ~[jasper.jar:9.0.41]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733) ~[servlet-api.jar:4.0.FR]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:712) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationDispatcher.doInclude(ApplicationDispatcher.java:580) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationDispatcher.include(ApplicationDispatcher.java:516) ~[catalina.jar:9.0.41]
	at com.liferay.portlet.PortletRequestDispatcherImpl.include(PortletRequestDispatcherImpl.java:206) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.liferay.portal.struts.PortletRequestProcessor.doInclude(PortletRequestProcessor.java:344) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.liferay.portal.struts.PortletRequestProcessor.doForward(PortletRequestProcessor.java:316) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.dotcms.repackage.org.apache.struts.tiles.TilesRequestProcessor.processTilesDefinition(TilesRequestProcessor.java:239) ~[dot.struts-1.2.10_1.jar:?]
	at com.dotcms.repackage.org.apache.struts.tiles.TilesRequestProcessor.processForwardConfig(TilesRequestProcessor.java:302) ~[dot.struts-1.2.10_1.jar:?]
	at com.dotcms.repackage.com.oroad.stxx.plugin.StxxTilesRequestProcessor.processForwardConfig(StxxTilesRequestProcessor.java:134) ~[dot.stxx-1.3_3.jar:?]
	at com.liferay.portal.struts.PortletRequestProcessor.processForwardConfig(PortletRequestProcessor.java:203) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.dotcms.repackage.org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:241) ~[dot.struts-1.2.10_1.jar:?]
	at com.liferay.portal.struts.PortletRequestProcessor.process(PortletRequestProcessor.java:89) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.liferay.portlet.StrutsPortlet.include(StrutsPortlet.java:152) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.liferay.portlet.StrutsPortlet.doView(StrutsPortlet.java:104) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.dotcms.repackage.javax.portlet.GenericPortlet.doDispatch(GenericPortlet.java:235) ~[dot.portlet-1.0_2.jar:?]
	at com.dotcms.repackage.javax.portlet.GenericPortlet.render(GenericPortlet.java:163) ~[dot.portlet-1.0_2.jar:?]
	at com.liferay.portlet.ConcretePortletWrapper.render(ConcretePortletWrapper.java:114) ~[dotcms_21.12_9aa6ca2.jar:?]
	at org.apache.jsp.html.portal.layout_jsp._jspService(layout_jsp.java:630) ~[?:?]
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:71) ~[jasper.jar:9.0.41]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733) ~[servlet-api.jar:4.0.FR]
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:477) ~[jasper.jar:9.0.41]
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:385) ~[jasper.jar:9.0.41]
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:329) ~[jasper.jar:9.0.41]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733) ~[servlet-api.jar:4.0.FR]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:712) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationDispatcher.doInclude(ApplicationDispatcher.java:580) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationDispatcher.include(ApplicationDispatcher.java:516) ~[catalina.jar:9.0.41]
	at org.apache.jasper.runtime.JspRuntimeLibrary.include(JspRuntimeLibrary.java:954) ~[jasper.jar:9.0.41]
	at org.apache.jsp.html.portal.layout_005fportal_jsp._jspService(layout_005fportal_jsp.java:1611) ~[?:?]
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:71) ~[jasper.jar:9.0.41]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733) ~[servlet-api.jar:4.0.FR]
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:477) ~[jasper.jar:9.0.41]
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:385) ~[jasper.jar:9.0.41]
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:329) ~[jasper.jar:9.0.41]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733) ~[servlet-api.jar:4.0.FR]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53) ~[tomcat-websocket.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:712) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationDispatcher.processRequest(ApplicationDispatcher.java:459) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationDispatcher.doForward(ApplicationDispatcher.java:384) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationDispatcher.forward(ApplicationDispatcher.java:312) ~[catalina.jar:9.0.41]
	at com.liferay.portal.struts.StrutsUtil.forward(StrutsUtil.java:72) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.liferay.portal.struts.PortalRequestProcessor.doForward(PortalRequestProcessor.java:161) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.dotmarketing.struts.PortalRequestProcessor.doForward(PortalRequestProcessor.java:48) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.dotcms.repackage.org.apache.struts.tiles.TilesRequestProcessor.processTilesDefinition(TilesRequestProcessor.java:239) ~[dot.struts-1.2.10_1.jar:?]
	at com.dotcms.repackage.org.apache.struts.tiles.TilesRequestProcessor.processForwardConfig(TilesRequestProcessor.java:302) ~[dot.struts-1.2.10_1.jar:?]
	at com.dotcms.repackage.com.oroad.stxx.plugin.StxxTilesRequestProcessor.processForwardConfig(StxxTilesRequestProcessor.java:134) ~[dot.stxx-1.3_3.jar:?]
	at com.dotcms.repackage.org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:241) ~[dot.struts-1.2.10_1.jar:?]
	at com.liferay.portal.struts.PortalRequestProcessor.process(PortalRequestProcessor.java:147) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.dotcms.repackage.org.apache.struts.action.ActionServlet.process(ActionServlet.java:1202) ~[dot.struts-1.2.10_1.jar:?]
	at com.dotcms.repackage.org.apache.struts.action.ActionServlet.doGet(ActionServlet.java:412) ~[dot.struts-1.2.10_1.jar:?]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:626) ~[servlet-api.jar:4.0.FR]
	at com.liferay.portal.servlet.MainServlet.callParentService(MainServlet.java:244) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.liferay.portal.servlet.MainServlet.service(MainServlet.java:444) ~[dotcms_21.12_9aa6ca2.jar:?]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733) ~[servlet-api.jar:4.0.FR]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53) ~[tomcat-websocket.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at com.dotcms.repackage.com.liferay.filters.secure.SecureFilter.doFilter(SecureFilter.java:135) ~[dot.secure-filter-ukv_2.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at com.dotmarketing.filters.CMSFilter.doFilterInternal(CMSFilter.java:168) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.dotmarketing.filters.CMSFilter.doFilter(CMSFilter.java:49) ~[dotcms_21.12_9aa6ca2.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at com.dotcms.filters.interceptor.AbstractWebInterceptorSupportFilter.doFilter(AbstractWebInterceptorSupportFilter.java:90) ~[dotcms_21.12_9aa6ca2.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at com.dotcms.filters.interceptor.AbstractWebInterceptorSupportFilter.doFilter(AbstractWebInterceptorSupportFilter.java:90) ~[dotcms_21.12_9aa6ca2.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at com.dotcms.vanityurl.filters.VanityURLFilter.doFilter(VanityURLFilter.java:114) ~[dotcms_21.12_9aa6ca2.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at org.tuckey.web.filters.urlrewrite.RuleChain.handleRewrite(RuleChain.java:176) ~[urlrewritefilter-4.0.4.jar:4.0.4]
	at org.tuckey.web.filters.urlrewrite.RuleChain.doRules(RuleChain.java:145) ~[urlrewritefilter-4.0.4.jar:4.0.4]
	at org.tuckey.web.filters.urlrewrite.UrlRewriter.processRequest(UrlRewriter.java:92) ~[urlrewritefilter-4.0.4.jar:4.0.4]
	at org.tuckey.web.filters.urlrewrite.UrlRewriteFilter.doFilter(UrlRewriteFilter.java:389) ~[urlrewritefilter-4.0.4.jar:4.0.4]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at com.dotmarketing.filters.TimeMachineFilter.doFilter(TimeMachineFilter.java:140) ~[dotcms_21.12_9aa6ca2.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at com.dotmarketing.filters.ThreadNameFilter.doFilter(ThreadNameFilter.java:88) ~[dotcms_21.12_9aa6ca2.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at com.dotmarketing.filters.CookiesFilter.doFilter(CookiesFilter.java:50) ~[dotcms_21.12_9aa6ca2.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at com.dotmarketing.filters.CharsetEncodingFilter.doFilter(CharsetEncodingFilter.java:99) ~[dotcms_21.12_9aa6ca2.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at com.dotcms.filters.interceptor.AbstractWebInterceptorSupportFilter.doFilter(AbstractWebInterceptorSupportFilter.java:90) ~[dotcms_21.12_9aa6ca2.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at org.apache.catalina.filters.HttpHeaderSecurityFilter.doFilter(HttpHeaderSecurityFilter.java:126) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at com.dotcms.filters.NormalizationFilter.doFilter(NormalizationFilter.java:73) ~[dotcms_21.12_9aa6ca2.jar:?]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97) ~[catalina.jar:9.0.41]
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143) ~[catalina.jar:9.0.41]
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92) ~[catalina.jar:9.0.41]
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:690) ~[catalina.jar:9.0.41]
	at org.apache.catalina.valves.RemoteIpValve.invoke(RemoteIpValve.java:764) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78) ~[catalina.jar:9.0.41]
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343) ~[catalina.jar:9.0.41]
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:374) ~[tomcat-coyote.jar:9.0.41]
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65) ~[tomcat-coyote.jar:9.0.41]
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:888) ~[tomcat-coyote.jar:9.0.41]
	at org.apache.tomcat.util.net.Nio2Endpoint$SocketProcessor.doRun(Nio2Endpoint.java:1693) ~[tomcat-coyote.jar:9.0.41]
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49) ~[tomcat-coyote.jar:9.0.41]
	at org.apache.tomcat.util.net.AbstractEndpoint.processSocket(AbstractEndpoint.java:1122) ~[tomcat-coyote.jar:9.0.41]
	at org.apache.tomcat.util.net.Nio2Endpoint.setSocketOptions(Nio2Endpoint.java:335) ~[tomcat-coyote.jar:9.0.41]
	at org.apache.tomcat.util.net.Nio2Endpoint$Nio2Acceptor.completed(Nio2Endpoint.java:455) ~[tomcat-coyote.jar:9.0.41]
	at org.apache.tomcat.util.net.Nio2Endpoint$Nio2Acceptor.completed(Nio2Endpoint.java:391) ~[tomcat-coyote.jar:9.0.41]
	at sun.nio.ch.Invoker.invokeUnchecked(Invoker.java:127) ~[?:?]
	at sun.nio.ch.Invoker.invokeDirect(Invoker.java:158) ~[?:?]
	at sun.nio.ch.Invoker.invoke(Invoker.java:186) ~[?:?]
	at sun.nio.ch.UnixAsynchronousServerSocketChannelImpl.onEvent(UnixAsynchronousServerSocketChannelImpl.java:193) ~[?:?]
	at sun.nio.ch.KQueuePort$EventHandlerTask.run(KQueuePort.java:312) ~[?:?]
	at sun.nio.ch.AsynchronousChannelGroupImpl$1.run(AsynchronousChannelGroupImpl.java:112) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
	at java.lang.Thread.run(Thread.java:834) [?:?]
Caused by: org.apache.jasper.JasperException: An exception occurred processing [/html/portlet/ext/contentlet/field/edit_field.jsp] at line [505]

502:         //IMAGE kind of field rendering
503:         else if (field.getFieldType().equals(
504:                 Field.FieldType.IMAGE.toString())) {
505:             final List<String> defaultPathFolderPathIds = BrowserUtil.getDefaultPathFolderPathIds(
506:                     contentlet, LegacyFieldTransformer.from(field),
507:                     user);
508:             defaultPathFolderPathIds.add(0, "root");


Stacktrace:
	at org.apache.jasper.servlet.JspServletWrapper.handleJspException(JspServletWrapper.java:626) ~[jasper.jar:9.0.41]
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:515) ~[jasper.jar:9.0.41]
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:385) ~[jasper.jar:9.0.41]
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:329) ~[jasper.jar:9.0.41]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733) ~[servlet-api.jar:4.0.FR]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:712) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationDispatcher.doInclude(ApplicationDispatcher.java:580) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationDispatcher.include(ApplicationDispatcher.java:516) ~[catalina.jar:9.0.41]
	at org.apache.jasper.runtime.JspRuntimeLibrary.include(JspRuntimeLibrary.java:954) ~[jasper.jar:9.0.41]
	at org.apache.jsp.html.portlet.ext.contentlet.edit_005fcontentlet_jsp._jspService(edit_005fcontentlet_jsp.java:4699) ~[?:?]
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:71) ~[jasper.jar:9.0.41]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733) ~[servlet-api.jar:4.0.FR]
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:477) ~[jasper.jar:9.0.41]
	... 153 more
Caused by: java.lang.NullPointerException
	at com.dotmarketing.portlets.browser.BrowserUtil$FolderPath.getIds(BrowserUtil.java:392) ~[dotcms_21.12_9aa6ca2.jar:?]
	at com.dotmarketing.portlets.browser.BrowserUtil.getDefaultPathFolderPathIds(BrowserUtil.java:356) ~[dotcms_21.12_9aa6ca2.jar:?]
	at org.apache.jsp.html.portlet.ext.contentlet.field.edit_005ffield_jsp._jspService(edit_005ffield_jsp.java:1370) ~[?:?]
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:71) ~[jasper.jar:9.0.41]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733) ~[servlet-api.jar:4.0.FR]
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:477) ~[jasper.jar:9.0.41]
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:385) ~[jasper.jar:9.0.41]
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:329) ~[jasper.jar:9.0.41]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733) ~[servlet-api.jar:4.0.FR]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationDispatcher.invoke(ApplicationDispatcher.java:712) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationDispatcher.doInclude(ApplicationDispatcher.java:580) ~[catalina.jar:9.0.41]
	at org.apache.catalina.core.ApplicationDispatcher.include(ApplicationDispatcher.java:516) ~[catalina.jar:9.0.41]
	at org.apache.jasper.runtime.JspRuntimeLibrary.include(JspRuntimeLibrary.java:954) ~[jasper.jar:9.0.41]
	at org.apache.jsp.html.portlet.ext.contentlet.edit_005fcontentlet_jsp._jspService(edit_005fcontentlet_jsp.java:4699) ~[?:?]
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:71) ~[jasper.jar:9.0.41]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733) ~[servlet-api.jar:4.0.FR]
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:477) ~[jasper.jar:9.0.41]
	... 153 more
19:36:25.883  ERROR portlet.StrutsPortlet - org.apache.jasper.JasperException: An exception occurred processing [/html/portlet/ext/contentlet/field/edit_field.jsp] at line [505]
```

--- a/dotCMS/src/integration-test/java/com/dotmarketing/portlets/browser/BrowserUtilTest.java
+++ b/dotCMS/src/integration-test/java/com/dotmarketing/portlets/browser/BrowserUtilTest.java
@@ -130,6 +130,42 @@ public class BrowserUtilTest {
 
     /**
      * Method to test: {@link BrowserUtil#getDefaultPathFolder(Contentlet, Field, User)}
+     * When: The {@link Field} has a wrong defaultPath's Field Variable
+     * Should: Return the folder set in the defaultPath
+     *
+     * @throws DotDataException
+     * @throws IOException
+     * @throws DotSecurityException
+     */
+    @Test
+    public void defaultPathEqualsToWrongFieldVariable() throws DotDataException, DotSecurityException {
+
+        final Map<String, Object> map = createContentletWithImageFieldWithoutValue();
+        addFolderHostField((ContentType) map.get(CONTENT_TYPE), (Contentlet) map.get(CONTENTLET));
+
+        final Host host = (Host) map.get(HOST);
+
+        final HttpServletRequest request = mock(HttpServletRequest.class);
+        HttpServletRequestThreadLocal.INSTANCE.setRequest(request);
+
+        final HttpSession httpSession = mock(HttpSession.class);
+        when(request.getSession(false)).thenReturn(httpSession);
+        when(request.getAttribute(Host.HOST_VELOCITY_VAR_NAME)).thenReturn(host.getIdentifier());
+
+        final Folder selectAsLastFolder = selectAsLastFolder(httpSession);
+
+        final Field field = (Field) map.get(FIELD);
+
+        addDefaultPathFieldVariable(host, field, "wrong_path");
+
+        final Optional<Folder> defaultPathFolder = BrowserUtil.getDefaultPathFolder(
+                (Contentlet) map.get(CONTENTLET), field, APILocator.systemUser());
+
+        assertEquals(selectAsLastFolder.getIdentifier(), defaultPathFolder.get().getIdentifier());
+    }
+
+    /**
+     * Method to test: {@link BrowserUtil#getDefaultPathFolder(Contentlet, Field, User)}
      * When: The {@link ContentType} has an Image field without value and a Host/Folder field
      * Should: Return the folder from the Contentlet's Host/Folder field value
      *

--- a/dotCMS/src/integration-test/java/com/dotmarketing/portlets/browser/BrowserUtilTest.java
+++ b/dotCMS/src/integration-test/java/com/dotmarketing/portlets/browser/BrowserUtilTest.java
@@ -120,12 +120,13 @@ public class BrowserUtilTest {
 
         final Field field = (Field) map.get(FIELD);
 
-        addDefaultPathFieldVariable(host, field, folder.getPath());
+        final Folder defaultPathFolderValue = new FolderDataGen().nextPersisted();
+        addDefaultPathFieldVariable(host, field, defaultPathFolderValue.getPath());
 
         final Optional<Folder> defaultPathFolder = BrowserUtil.getDefaultPathFolder(
                 (Contentlet) map.get(CONTENTLET), field, APILocator.systemUser());
 
-        assertEquals(folder.getIdentifier(), defaultPathFolder.get().getIdentifier());
+        assertEquals(defaultPathFolderValue.getIdentifier(), defaultPathFolder.get().getIdentifier());
     }
 
     /**
@@ -144,6 +145,7 @@ public class BrowserUtilTest {
         addFolderHostField((ContentType) map.get(CONTENT_TYPE), (Contentlet) map.get(CONTENTLET));
 
         final Host host = (Host) map.get(HOST);
+        final Folder folder = (Folder) map.get(FOLDER);
 
         final HttpServletRequest request = mock(HttpServletRequest.class);
         HttpServletRequestThreadLocal.INSTANCE.setRequest(request);
@@ -161,7 +163,7 @@ public class BrowserUtilTest {
         final Optional<Folder> defaultPathFolder = BrowserUtil.getDefaultPathFolder(
                 (Contentlet) map.get(CONTENTLET), field, APILocator.systemUser());
 
-        assertEquals(selectAsLastFolder.getIdentifier(), defaultPathFolder.get().getIdentifier());
+        assertEquals(folder.getIdentifier(), defaultPathFolder.get().getIdentifier());
     }
 
     /**

--- a/dotCMS/src/main/java/com/dotmarketing/portlets/browser/BrowserUtil.java
+++ b/dotCMS/src/main/java/com/dotmarketing/portlets/browser/BrowserUtil.java
@@ -111,13 +111,12 @@ public class BrowserUtil {
             List<FieldVariable> fieldVariables = APILocator.getFieldAPI()
                     .getFieldVariablesForField(field.id(), user, true);
 
-            final List<FieldVariable> defaulPathVariable = fieldVariables.stream()
+            final Optional<FieldVariable> defaulPathVariable = fieldVariables.stream()
                     .filter(fieldVariable -> "defaultPath".equals(fieldVariable.getKey()))
-                    .limit(1)
-                    .collect(Collectors.toList());
+                    .findFirst();
 
-            if (UtilMethods.isSet(defaulPathVariable)) {
-                final FieldVariable defaultPathVariable = defaulPathVariable.get(0);
+            if (defaulPathVariable.isPresent()) {
+                final FieldVariable defaultPathVariable = defaulPathVariable.get();
                 final String fieldVariableValue = defaultPathVariable.getValue();
 
                 if (fieldVariableValue.startsWith(HOST_INDICATOR)) {
@@ -141,11 +140,13 @@ public class BrowserUtil {
                 } else {
                     final String currentHost = WebAPILocator.getHostWebAPI().getCurrentHost()
                             .getIdentifier();
-                    return Optional.of(
-                        APILocator.getFolderAPI()
+
+                    final Folder folderByPath = APILocator.getFolderAPI()
                             .findFolderByPath(fieldVariableValue, currentHost,
-                                    APILocator.systemUser(), false)
-                    );
+                                    APILocator.systemUser(), false);
+
+                    return UtilMethods.isSet(folderByPath) && UtilMethods.isSet(folderByPath.getIdentifier())
+                        ? Optional.of(folderByPath) : Optional.empty();
                 }
             } else {
                 Logger.warn(BrowserUtil.class, () -> "defaultPath variable not exists for field " + field.name());

